cypress/support/step_definitions/index.ts

```diff
@@ -77,10 +77,12 @@ When(/^I click the "(.*)" element(?: and select "(.*)")?$/, (selector: string, v
 })
 
 When('I {string} {string} into the {string} element', (event: string, text: string, selector: string) => {
+  const method = text === '' ? 'clear' : 'type'
```

Comment:
nit:

`And I clear {string} into the "input" element`

Sounds a bit weird, also at second glance is that even right? Anyway, is there anyway we can make this read better, either regex acrobatics or a separate step would be fine.

Reply:
Oh, no. The intended usage of this would be:

> And I type "" into the "$input" element

But I now realize that that’s ambiguous because it could mean "I add the empty string to the end of input". Maybe it should just be a separate step.

Reply:
I’ve changed this to a separate step now.